test/nodes/test_file_converter.py

```diff
@@ -296,21 +296,21 @@ def test_markdown_converter_headline_extraction():
 
 
 @pytest.mark.unit
-def test_markdown_converter_frontmatter_to_meta():
+def test_markdown_converter_frontmatter_to_meta(samples_path):
     converter = MarkdownConverter(add_frontmatter_to_meta=True)
-    document = converter.convert(file_path=SAMPLES_PATH / "markdown" / "sample.md")[0]
+    document = converter.convert(file_path=samples_path / "markdown" / "sample.md")[0]
     assert document.meta["type"] == "intro"
     assert document.meta["date"] == "1.1.2023"
 
 
 @pytest.mark.unit
-def test_markdown_converter_remove_code_snippets():
+def test_markdown_converter_remove_code_snippets(samples_path):
```

Comment:
It's confusing that the test uses `remove_code_snippets=False` but its name is `test_markdown_converter_remove_code_snippets`